.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
+labels: bug
+---
+
+<!-- Lines like this are comments and will be invisible -->
+
+# Bug / performance issue / build issue
```

Comment:
is the person writing the ticket meant to pick one of these and delete the others?

Reply:
if they want to - sure. if they don't - fine with me as well.